packages/theme/src/theme.ts

```diff
@@ -122,15 +135,151 @@ export interface Theme {
     primary: string;
     secondary: string;
     focus: string;
-    radius: typeof borderRadius;
-    width: typeof borderWidth;
+    radius: {
+      small: number;
+      regular: number;
+      circle: number;
+    };
+    width: {
+      slim: number;
+      medium: number;
+    };
   };
   icon: {
-    size: typeof iconSizes;
+    size: {
+      xSmall: number;
+      small: number;
+      normal: number;
+      large: number;
+    }
   };
   geofencingZones: GeofencingZoneStyles;
 }
 
+export type ContrastColorFs = {
```

Comment:
What does `Fs` mean here? Intuitively I think of file-system when I see the fs abbreviation.

Reply:
The idea is `Figma Structure`, so I ended up with `FS` with a lack of a better abbreviation...